src/transforms/filter.jl

```diff
@@ -69,22 +65,48 @@ DropMissing(cols::T...) where {T<:ColSelector} =
 
 isrevertible(::Type{<:DropMissing}) = true
 
-_ftrans(::DropMissing{Colon}, table) =
+_ftrans(::DropMissing{Colon}, cols) =
   Filter(row -> all(!ismissing, row))
 
-function _ftrans(transform::DropMissing, table)
-  allcols = Tables.columnnames(table)
-  cols = _filter(transform.colspec, allcols)
+_ftrans(::DropMissing, cols) =
   Filter(row -> all(!ismissing, getindex.(Ref(row), cols)))
-end
+
+# nonmissing 
+_nonmissing(::Type{T}, x) where {T} = x
+_nonmissing(::Type{Union{Missing,T}}, x) where {T} = collect(T, x)
+_nonmissing(x) = _nonmissing(eltype(x), x)
 
 function apply(transform::DropMissing, table)
-  ftrans = _ftrans(transform, table)
+  names = Tables.columnnames(table)
```

Comment:
given the other names in this function, maybe we should also name this colnames

Reply:
or maybe rename coltypes --> types and selnames --> selection

Reply:
OK

Reply:
Made in [3634072](https://github.com/JuliaML/TableTransforms.jl/pull/35/commits/36340728613133dd12ca9c6c27a04e86f9d09c1c)